mms/docker-compose.yml

```diff
@@ -1,35 +1,50 @@
-version: "3"
 
 services:
-
   scylla-node1:
     container_name: scylla-node1
-    image: scylladb/scylla:5.2.0
+    image: scylladb/scylla
```

Comment:
Please use a specific ScyllaDB version in all Docker files (not just here). You can use 2025.3, which is the latest